### ISSUE TITLE
Bug: NoData and Loading text are shown at the same time

### ISSUE THREAD

**Issue by bay-jacewhitten** [komarovalexander/ka-table#303]
Both the no data and the loading messages are shown at the same time. 

https://stackblitz.com/edit/table-loading-ts-9dhmvb?file=Demo.tsx

**Comment by komarovalexander** [comment]
Hi @bay-jacewhitten thanks for the issue
this is workaround:  
```
        childComponents={{
          noDataCell: {
            content: ({ noData }) =>
              option.loading.enabled ? <></> : noData?.text,
          },
        }}
```
https://stackblitz.com/edit/table-loading-ts-q9xejf?file=Demo.tsx,node_modules%2Fka-table%2FModels%2FChildComponents.d.ts,node_modules%2Fka-table%2Fprops.d.ts

**Comment by komarovalexander** [comment]
fixed in v7.14.1 https://stackblitz.com/edit/table-loading-ts-9dhmvb?file=Demo.tsx
